DATA/EQUIPMENT/weapon_equip.ini

```diff
@@ -32920,8 +32920,8 @@ ids_info = 461294
 DA_archetype = equipment\models\weapons\li_cannon.cmp
 material_library = equipment\models\li_equip.mat
 HP_child = HPConnect
-hit_pts = 2632
-explosion_rsisteance = 4.000000
+hit_pts = 1500
+explosion_rsisteance = 0.2

```

Comment:
They key "explosion_rsisteance" has a typo.

Reply:
I noted that in my sheet but still forgot to polish it in review. Sloppy. Thanks.